python/tests/compat/test_mantid.py

```diff
@@ -134,7 +134,28 @@ def test_unit_conversion(self):
                     converted_mantid.coords['wavelength'].values,
                     converted.coords['wavelength'].values,
                 )))
-        # delta = sc.sum(converted_mantid - converted, 'spectrum')
+
+    def test_inelastic_unit_conversion(self):
+        import mantid.simpleapi as mantid
+        eventWS = self.base_event_ws
+        ws_deltaE = mantid.ConvertUnits(eventWS,
+                                        Target='DeltaE',
+                                        EMode='Direct',
+                                        EFixed=3)
+        ref = mantidcompat.from_mantid(ws_deltaE)
+        da = mantidcompat.from_mantid(eventWS)
+        da.coords['incident-energy'] = 3.0 * sc.units.meV
+        da = sc.neutron.convert(da, 'tof', 'energy-transfer')
+        assert sc.all(
+            sc.isnan(da.coords['energy-transfer']) | sc.is_approx(
+                da.coords['energy-transfer'], ref.coords['energy-transfer'],
+                1e-8 * sc.units.meV +
+                1e-8 * sc.abs(ref.coords['energy-transfer']))).value
+        assert sc.all(
+            sc.isnan(da.bins.data.coords['energy-transfer']) | sc.is_approx(
+                da.bins.data.coords['energy-transfer'],
+                ref.bins.data.coords['energy-transfer'], 1e-8 * sc.units.meV +
+                1e-8 * sc.abs(ref.bins.data.coords['energy-transfer']))).value
```

Comment:
👍 , but no equivalent cross-check for indirect conversion?

Reply:
Yes, we should add that. Can you think of an indirect instrument we have in the Mantid data repo?

Reply:
Not off the top of my head. @Matthew-Andrew would surely be able to recall one as he worked in that area.